src/apps/tco2_dashboard/retirement_trends/impl/retirement_trends_pool.py

```diff
@@ -411,6 +414,54 @@ def modify_klima_token_retirements_df(self, df):
         )
         df['View on PolygonScan'] = '[Click Here](' + df['View on PolygonScan'] + ')'
 
+        df['Project_num'] = df['Project'].apply(
+            lambda x: pd.Series(str(x).split("-"))
+            )[1]
+
+        df.Project_num.fillna('N/A', inplace=True)
+
+        verra_l = 'https://registry.verra.org/app/projectDetail/VCS/'
+
+        df['Project_Link'] = verra_l
+
+        df["Project_Link"] = df["Project_Link"] + df["Project_num"]
+
+        missing_condition_1 = df['Project_Link'].str.match(
+            'https://registry.verra.org/app/projectDetail/VCS/N/A')
+
+        df['Project_Link'] = np.where(
+            missing_condition_1, "N/A", df['Project_Link']
+            )
+
+        missing_condition_2 = df['Project_num'].str.match("N/A")
+
+        df['Project_Link'] = np.where(
+            missing_condition_2,
+            'N/A',
+            "[" + df['Project'] + "]" + "(" + df['Project_Link'] + ")"
+            )
+
+        pools_condition = df['Pool'].str.match(
+            "BCT|NBO|NCT|UBO|MCO2|0x0000000000000000000000000000000000000000")
+
+        df['Pool'] = np.where(pools_condition, df['Pool'], 'N/A')
```

Comment:
I think the match is required as the idea of this is to keep all the pool retirements and the 0x0 strings, and set any TCO2 strings to NA. I think we concluded in the issue [comments](https://github.com/KlimaDAO/dash-apps/issues/122#issuecomment-1535549807) that 0x0 were MCO2 retirements that had an issue in the subgraph, and we wanted to leave them as 0x0 for now? (I may be interpreting your 'ignore them' as leave as is instead of set to n/a though)